lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_facts.py

```diff
@@ -65,7 +65,7 @@
 '''
 
 RETURN = '''
-vms:
+azure_virtualmachine:
```

Comment:
You should never change the return values of an existing module. This will break backwards compatibility.